app/services/healthcheck.rb

```diff
@@ -0,0 +1,60 @@
+require 'sidekiq/api'
+
+class Healthcheck
```

Comment:
That sounds like a good idea. Were you thinking of something like this:

```ruby
def checks
  {
    # ...
   }
rescue
  { all: 'broken' }
end
```

or trapping each of the key methods (e.g. `database_active?`; `old_items?` might need its logic reversed) to return `false` on error?